From kraxel@bytesex.org Wed Jan 21 17:13:31 2004
Return-path: <linux-kernel-owner+arekm=40pld-linux.org@vger.kernel.org>
Envelope-to: arekm@sse.pl
Delivery-date: Wed, 21 Jan 2004 17:36:49 +0100
Received: from virtual.perfopol.pl [213.25.186.10]
	by localhost with POP3 (fetchmail-6.2.5)
	for arekm@localhost (single-drop); Wed, 21 Jan 2004 17:40:11 +0100 (CET)
Received: from gabber.metalab.unc.edu ([152.2.241.57])
	by mail.perfopol.pl with smtp (Exim 4.24)
	id 1AjLLQ-0004x6-8O
	for arekm@sse.pl; Wed, 21 Jan 2004 17:36:44 +0100
Received: (qmail 10623 invoked by uid 1012); 21 Jan 2004 16:36:43 -0000
Delivered-To: pld-arekm@pld-linux.org
Received: (qmail 10619 invoked from network); 21 Jan 2004 16:36:43 -0000
Received: from vger.kernel.org (67.72.78.212)
  by gabber.metalab.unc.edu with SMTP; 21 Jan 2004 16:36:43 -0000
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
	id S265728AbUAUQZU (ORCPT <rfc822;arekm@pld-linux.org>);
	Wed, 21 Jan 2004 11:25:20 -0500
Received: (majordomo@vger.kernel.org) by vger.kernel.org id S265744AbUAUQZU
	(ORCPT <rfc822;linux-kernel-outgoing>);
	Wed, 21 Jan 2004 11:25:20 -0500
Received: from hirsch.in-berlin.de ([192.109.42.6]:31126 "EHLO
	hirsch.in-berlin.de") by vger.kernel.org with ESMTP id S265728AbUAUQZQ
	(ORCPT <rfc822;linux-kernel@vger.kernel.org>);
	Wed, 21 Jan 2004 11:25:16 -0500
X-Envelope-From: kraxel@bytesex.org
Received: from hirsch.in-berlin.de (localhost [127.0.0.1])
	by hirsch.in-berlin.de (8.12.10/8.12.10/Debian-6) with ESMTP id i0LGPBcl010646;
	Wed, 21 Jan 2004 17:25:11 +0100
Received: (from uucp@localhost)
	by hirsch.in-berlin.de (8.12.10/8.12.10/Debian-5) with UUCP id i0LGPBAR010642;
	Wed, 21 Jan 2004 17:25:11 +0100
Received: from bytesex.org (kraxel@localhost [127.0.0.1])
	by bytesex.org (8.12.3/8.12.3/Debian-6.6) with ESMTP id i0LGDVnW002537;
	Wed, 21 Jan 2004 17:13:31 +0100
Received: (from kraxel@localhost)
	by bytesex.org (8.12.3/8.12.3/Debian-6.6) id i0LGDVqY002536;
	Wed, 21 Jan 2004 17:13:31 +0100
Date:	Wed, 21 Jan 2004 17:13:31 +0100
From:	Gerd Knorr <kraxel@bytesex.org>
To: Linus Torvalds <torvalds@osdl.org>,
 Andrew Morton <akpm@osdl.org>,
 Kernel List <linux-kernel@vger.kernel.org>
Subject: [patch] selinux build fix
Message-ID: <20040121161331.GA2531@bytesex.org>
Mime-Version: 1.0
Content-Type: text/plain;
  charset=us-ascii
Content-Disposition: inline
User-Agent: Mutt/1.5.3i
Sender:	linux-kernel-owner@vger.kernel.org
Precedence: bulk
X-Mailing-List:	linux-kernel@vger.kernel.org
Status: R
X-Status: N
X-KMail-EncryptionState:  
X-KMail-SignatureState:  
X-KMail-MDN-Sent:  

  Hi

trivial one: uses __init and thus needs linux/init.h

  Gerd

--- linux/security/selinux/netif.c.se	2004-01-21 16:42:10.000000000 +0100
+++ linux/security/selinux/netif.c	2004-01-21 16:42:22.000000000 +0100
@@ -12,6 +12,7 @@
  * it under the terms of the GNU General Public License version 2,
  * as published by the Free Software Foundation.
  */
+#include <linux/init.h>
 #include <linux/types.h>
 #include <linux/stddef.h>
 #include <linux/kernel.h>

-- 
"... und auch das ganze Wochenende oll" -- Wetterbericht auf RadioEins
-
To unsubscribe from this list: send the line "unsubscribe linux-kernel" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html
Please read the FAQ at  http://www.tux.org/lkml/


